ecyp-database.Rmd

```diff
@@ -150,10 +165,13 @@ output$UI <- renderUI({
 
   ecyp_db <- refresh_table(sheet_url)
 
-  variable_list <- c("---", setdiff(names(ecyp_db), c("Unique_Number", "Last_Updated")))
+  variable_list <- c("---", setdiff(names(ecyp_db), c("Unique Number", "Last Updated")))
```

Comment:
Got rid of underscores in data.frame that is shown to user so that means the column names also have to change to spaces instead of underscores.